### PR TITLE
feat: added list component

### PR DESCRIPTION
**Pull request by mansi2011**
list

feat: added list component
- spaced
- stacked

### CODE REVIEW

File: Components/List/List.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.spaced-list-style{
+    /* text-decoration: none; */
+    display: inline;
+    padding: 3rem 8rem;
+    font-size: 2rem;
+}
+
+.stacked-list-style{
+    /* text-decoration: none; */
```
Comment (Hardiegogo) [inline Components/List/List.css]: Remove unused code.

**Review by Hardiegogo — APPROVED** [review]
Looks nice, you can merge it after taking into consideration some suggested changes.

File: Components/List/List.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.spaced-list-style{
+    /* text-decoration: none; */
+    display: inline;
+    padding: 3rem 8rem;
```
Comment (Prabhav-Sharma) [inline Components/List/List.css:3]: If you use these spacings regularly, then you can define a global variable. Otherwise, it is fine.

Comment (mansi2011) [reply Components/List/List.css:3]: this was unique combination applied for first time else i have define variables for common combination

File: Components/List/List.css (hunk 1 above)
Comment (mansi2011) [reply Components/List/List.css]: done somehow i missed that

**Pull request merged**